pages.fr/linux/usermod.md

```diff
@@ -1,16 +1,25 @@
 # usermod
 
-> Modifie un compte utilisateur.
+> Modifie le compte d'un utilisateur.
+> Voir aussi `users`, `useradd`, `userdel`.
 > Plus d'informations : <https://manned.org/usermod>.
 
 - Change le nom d'un utilisateur :
 
-`usermod -l {{nouveau_nom}} {{utilisateur}}`
+`sudo usermod --login {{nouveau_nom}} {{nom_utilisateur}}`
+
+- Modifie l'identifiant numérique d'un utilisateur :
+
+`sudo usermod --uid {{identifiant}} {{nom_utilisateur}}`
+
+- Change le shell d'un utilisateur :
+
+`sudo usermod --shell {{chemin/vers/shell}} {{nom_utilisateur}}`
 
 - Ajoute l'utilisateur à des groupes supplémentaires (attention à l'omission d'espaces) :
 
-`usermod -a -G {{groupe1,groupe2}} {{utilisateur}}`
+`sudo usermod --append --groups {{groupe1,groupe2,...}} {{nom_utilisateur}}`
 
-- Crée un nouveau dossier home pour un utilisateur et déplace ses fichiers vers celui-ci :
+- Change le répertoire personnel d'un utilisateur et déplace ses fichiers vers celui-ci :
 
-`usermod -m -d {{/chemin/vers/home}} {{utilisateur}}`
+`sudo usermod --move-home --home {{chemin/vers/nouveau/répertoire}} {{nom_utilisateur}}`
```

Comment:
```suggestion
`sudo usermod --move-home --home {{chemin/vers/nouveau_répertoire}} {{nom_utilisateur}}`
```

Optional fix (the [original](https://github.com/tldr-pages/tldr/blob/main/pages/linux/usermod.md) page can be updated to reflect this change): `_` can be used instead of `/` to be synced to the original example.